.github/workflows/pull-request.yaml

```diff
@@ -35,9 +35,7 @@ jobs:
           prettier_command_prefix: yarn
 
       - name: Test
-        run: yarn run jest --ci --reporters github-actions --reporters summary --coverage .
-        env:
-          NODE_OPTIONS: '--no-warnings --experimental-vm-modules'
+        run: yarn run test  --reporter github-actions --reporter default  --coverage
```

Comment:
**suggestion:** Remove reporters configuration

`github-actions` is the default [when running in CI](https://vitest.dev/guide/reporters.html#github-actions-reporter) and `default` is well, the default.

Reply:
Oops, missed it.